README.md

```diff
@@ -1,36 +1,282 @@
-High Five
-=========
+# High Five
 
-HTML5 build/deploy tool, usually used with PhoneGap but also for standalone web-based apps
+HTML5 build/deploy tool, usually used with PhoneGap but also for standalone web-based apps.
 
-Ruby
-====
+## Installation
 
-If you're on Windows, you need both Ruby and the Dev tools.  There is a great tutorial here: 
+Add `gem 'high_five'` to your `Gemfile` or run `gem install high_five`.
 
-https://github.com/oneclick/rubyinstaller/wiki/development-kit
+After installing the high\_five gem, run
+
+```
+$ hi5 init
+```
+
+This will bootstrap your project, creating a `config` directory if one isn't present and a few important files inside:
+
+* `config/high_five.rb`
+
+The most most important file. It is the main configuration file and where most configuration takes place. See below for configuration options. It contains sensible defaults.
+
+* `config/high_five/app-common.js`
+
+This is where your shared javascript goes.
+
+If you are using a framework like Sencha, this is where you would include the Sencha files.
+
+At a minimum, you will likely include the main entry point for your app here.
+
+This file is processed with Sprockets, so anything included here will be minified.
+
+Do not add library or remote scripts here. Those should be added with `config/high_five.rb` (see below for how to do this).
+
+* `config/high_five/app-platform.js`
+
+This file is a template and *must* be copied to create `app-<platform>.js` for each platform you have which by default are 'ios' and 'android'.
+
+After creating `app-<platform>.js` for each of your platforms, you can delete this file.
+
+TODO: Just create the app-ios.js and app-android.js files instead of app-platform.js. This would get us a little closer to "just working".
+
+* `config/high_five/index.html.erb`
+
+The entry page for your app.
+
+TODO: Why reverse on @stylesheets? Is there a better way to do it?
+
+TODO: Why not reverse on @javascripts? Should it be like @stylesheets or the other way around? Or are they correct the way there are now?
```

Comment:
They are correct, the javascripts we want to preserve the order that is generated by sprockets which has robust include order logic.  The stylesheets on the other had get pushed onto an array as they are called in the config.  It would actually probably be better to unshift them so we didnt need to reverse later but that's the logic